modules/v1/users/models/LoginForm.php

```diff
@@ -102,7 +102,7 @@ public function getActiveUser()
     {
         if ($this->_user === false) {
             $this->_user = User::find()
-                ->byEmailOrUsername($this->username)
+                ->byEmailOrUsername($this->email)
```

Comment:
@ll-bat Let's change this so that it only searches inside email on inside username